### PR TITLE
[stable8.1] Add yaml to file actions

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @felixboehm @LukasReschke @nickvergessen @oparoz 

Backport approval: https://github.com/owncloud/files_texteditor/pull/89#issuecomment-123676131

### CODE REVIEW

**Comment by MorrisJobke** [comment]
I tested this and it works :)

File: js/editor.js [hunk 1]
```diff
@@ -397,6 +397,10 @@ $(document).ready(function () {
 			showFileEditor($('#dir').val(), filename);
 		});
 		FileActions.setDefault('application/x-tex', 'Edit');
+		FileActions.register('application/yaml', 'Edit', OC.PERMISSION_READ, '', function (filename) {
+			showFileEditor($('#dir').val(), filename);
+		});
+		FileActions.setDefault('application/yaml', 'Edit');
```
Comment (nickvergessen) [inline js/editor.js]: Should this group be a method? :hide:

**Comment by nickvergessen** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@DeepDiver1975 This is a backport to stable8.1 ;)

File: js/editor.js (hunk 1 above)
Comment (MorrisJobke) [inline js/editor.js]: Not in a backport

**Pull request merged**